h2o-genmodel/src/main/java/hex/genmodel/attributes/metrics/MojoModelMetricsRegressionCoxPH.java

```diff
@@ -0,0 +1,8 @@
+package hex.genmodel.attributes.metrics;
+
+import hex.genmodel.attributes.SerializedName;
+
+public class MojoModelMetricsRegressionCoxPH extends MojoModelMetricsRegression {
+  @SerializedName("concordance")
+  public long concordance;  
```

Comment:
Why is this long?

Reply:
Is this class used?

Reply:
in `determineModelMetricsType()` in `ModelAttributes`